tests/legacy-cli/e2e/utils/tar.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import fs from 'fs';
+import { normalize } from 'path';
+import { Parse } from 'tar';
+
+/**
+ * Extract and return the contents of a single file out of a tar file.
+ *
+ * @param tarball the tar file to extract from
+ * @param filePath the path of the file to extract
+ * @returns the Buffer of file or an error on fs/tar error or file not found
+ */
+export async function extractFile(tarball: string, filePath: string): Promise<Buffer> {
+  return new Promise((resolve, reject) => {
+    let chunks: Buffer[] | null = null;
```

Comment:
Is there a reason that the chunks need defined in this block?  Since its only referenced inside of the `onentry` closure, you should be able to just define it there correct?

Reply:
It was probably just me learning the fs/parse API. Updated 👍